astroplan/tests/test_core.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord)
+import astropy.units as u
+from astropy.time import Time
+from astropy.tests.helper import remote_data
+import numpy as np
+import ephem
+import pytz
+
+from ..core import FixedTarget, Observer
+
+
+def test_Observer_constructor_location():
+    '''
+    Show that location defined by latitude/longitude/elevation is parsed
+    identically to passing in an `~astropy.coordinates.EarthLocation` directly.
+    '''
+
+    lat = '+19:00:00'
+    lon = '-155:00:00'
+    elevation = 0.0 * u.m
+    location = EarthLocation.from_geodetic(lon, lat, elevation)
+
+    environment_kwargs = dict(pressure=1*u.bar, relative_humidity=0.1,
+                              temperature=10*u.deg_C)
+
+    obs1 = Observer(name='Observatory',
+                    latitude=lat,
+                    longitude=lon,
+                    elevation=elevation,
+                    **environment_kwargs)
+
+    obs2 = Observer(name='Observatory',
+                    location=location,
+                    **environment_kwargs)
+
+    assert obs1.location == obs2.location, ('Locations not parsed equivalently')
+
+@remote_data
+def test_FixedTarget_from_name():
+    '''
+    Check that resolving target names with the `SkyCoord.from_name` constructor
+    to produce a `FixedTarget` accurately resolves the coordinates of Polaris.
+    '''
+
+    # Resolve coordinates with SkyCoord.from_name classmethod
+    polaris_from_name = FixedTarget.from_name('Polaris')
+    # Coordinates grabbed from SIMBAD
+    polaris_from_SIMBAD = SkyCoord('02h31m49.09456s', '+89d15m50.7923s')
+
+    # Make sure separation is small
+    assert polaris_from_name.coord.separation(polaris_from_SIMBAD) < 1*u.arcsec
+
+def test_Observer_altaz():
+    '''
+    Check that the altitude/azimuth computed by `Observer.altaz` is similar
+    to the result from PyEphem when pressure = 0 (no atmosphere) for Vega at
+    2000-01-01 12:00:00 UTC.
+    '''
+    # Define the test case
+    latitude = '00:00:00'
+    longitude = '00:00:00'
+    elevation = 0 # [m]
+    pressure = 0  # no atmosphere
+    time = Time('2000-01-01 12:00:00')
+    vega_coords = SkyCoord('18h36m56.33635s', '+38d47m01.2802s')
+
+    # Calculate altitude/azimuth with astroplan
+    location = EarthLocation.from_geodetic(longitude, latitude, elevation*u.m)
+    astroplan_obs = Observer(name='Observatory', location=location,
+                             pressure=pressure*u.bar)
+    astroplan_vega = FixedTarget(vega_coords)
+    altaz = astroplan_obs.altaz(astroplan_vega, time)
+    astroplan_altitude = altaz.alt
+    astroplan_azimuth = altaz.az
+
+    # Calculate altitude/azimuth with PyEphem
+    pyephem_obs = ephem.Observer()
```

Comment:
Just one more comment: the function that prints the reference results to the console could live right next to the actual test function in this file. If it's name doesn't start with `test_` it will not be collected as a test, but you can still import and execute it manually. So the reference results for `test_Observer_altaz` could be computed by `print_reference_Observer_altaz`. I think this would be my preference, but yes, let's see what the others think.